justfile

```diff
@@ -0,0 +1,24 @@
+update-lockfiles:
+    update-workspace-lockfile
+    update-template-lockfile
+
+update-workspace-lockfile:
+    cargo update
+
+update-template-lockfile:
+    #!/usr/bin/env bash
+    set -euxo pipefail
+    tmpdir=$(mktemp -d)
+
+    # special project name to make it appear at the end of the lockfile
```

Comment:
Why does it matter?

Reply:
It doesn't technically; I just want to make sure the position of the project entry is predictable and that it doesn't appear in the middle of the file for no reason.

Reply:
It's a lock file, it doesn't really matter at all

Reply:
Well, it matters for nicer git diffs, and for potential easier debugging of the template variables (now they're always at the end of a really long file, so much easier to find).